internal/access/web/sdk_e2e_multi_test.go

```
@@ -0,0 +1,120 @@
+client = Client(sys.argv[2])
+result = bus_journeys_search(client, BusJourneysSearchInput(
+    origin_stop_id="stop-a",
+    destination_stop_id="stop-b",
+    depart_after=datetime.now() - timedelta(days=400),
+))
+print(json.dumps(result))
+`
+	if err := os.WriteFile(filepath.Join(dir, "main.py"), []byte(main), 0644); err != nil {
+		t.Fatal(err)
+	}
+	command := exec.Command("python", "main.py", dir, testServer.URL)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)